### PR TITLE
[PBIOS-291] Currency Kit

### PR DESCRIPTION
**Pull request by RachelRadford21**
**What does this PR do?** A clear and concise description with your runway ticket url.
[PBIOS-291] Currency Kit

**Screenshots:** Screenshots to visualize your addition/change
Simulator Screenshot - iPhone 15 Pro Max - 2024-03-01 at 14 00 03

**How to test?** Steps to confirm the desired behavior:
4. See addition/change

### Checklist
- [x] **LABELS** - Add a label: `breaking`, `bug`, `improvement`, `documentation`, or `enhancement`. See [Labels](https://github.com/powerhome/playbook-apple/labels) for descriptions.
- [x] **RELEASES** - Add the appropriate label: `Ready for Testing` / `Ready for Release`
- [x] **TESTING** - Have you tested your story?

### CODE REVIEW

File: Sources/Playbook/Components/Currency/CurrencyCatalog.swift [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          PBDoc(title: "Size") {
+            sizeView
+          }
+          PBDoc(title: "Alignment") {
+            alignmentView
+          }
+        }
+        .padding(Spacing.medium)
+      }
+      .background(Color.background(Color.BackgroundColor.light))
```
Comment (isismsilva) [inline Sources/Playbook/Components/Currency/CurrencyCatalog.swift:25]:
```suggestion
      .background(Color.background(.light))
```

Comment (RachelRadford21) [reply Sources/Playbook/Components/Currency/CurrencyCatalog.swift]: Done!

File: Sources/Playbook/Components/Currency/PBCurrency.swift [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  }
+  var formattedDecimal: AttributedString {
+    return colorAttributedText(hasUnit ? unit ?? "" : decimalAmount ?? "", characterToChange: hasUnit ? unit ?? "" : decimalAmount ?? "", color: .text(.light))
+  }
+  var fontColor: Color {
+    isEmphasized ? .text(.default) : .text(.light)
+  }
+  func getCurrency(symbol: String) -> String {
+    let locale = Locale(identifier: symbol)
+    let currencySymbol = locale.currencySymbol!
```
Comment (isismsilva) [inline Sources/Playbook/Components/Currency/PBCurrency.swift:88]: I think this might break the app if the symbol doesn't exist

Comment (RachelRadford21) [reply Sources/Playbook/Components/Currency/PBCurrency.swift:88]: Good catch!

**Review by isismsilva — APPROVED** [review]

**Pull request merged**